rasa/model_training.py

```diff
@@ -516,9 +512,8 @@ async def _train_core_with_validated_data(
         rasa.shared.utils.cli.print_color(
             "Training Core model...", color=rasa.shared.utils.io.bcolors.OKBLUE
         )
-        domain, config = await asyncio.gather(
-            file_importer.get_domain(), file_importer.get_config()
-        )
+        domain = file_importer.get_domain()
```

Comment:
`_core_model_for_finetuning` also doesn't need to be `async` once the fingerprinting method isn't async any longer